src/java/fr/paris/lutece/plugins/notifygru/modules/forms/services/provider/FormsProviderManager.java

```diff
@@ -207,7 +207,19 @@ private Collection<InfoMarker> getProviderMarkerDescriptions( List<Form> listFor
         }
         InfoMarker notifyMarkerURl = new InfoMarker( Constants.MARK_URL_ADMIN_RESPONSE );
         notifyMarkerURl.setDescription( I18nService.getLocalizedString( Constants.MESSAGE_DESCRIPTION, I18nService.getDefaultLocale( ) ) );
+        InfoMarker creationDateMarker = new InfoMarker( Constants.MARK_CREATION_DATE );
+        creationDateMarker.setDescription( I18nService.getLocalizedString( Constants.MESSAGE_CREATION_DATE, I18nService.getDefaultLocale( ) ) );
```

Comment:
it would be better to send the I18n keys in the bookmarks, so that they are processed by the i18n processing which occurs after the freemarker processing of the template